generic-styles/index.less

```diff
@@ -1,3 +1,31 @@
 #main-content {
   @import "./content-baked.less";
 }
+#main-content > .os-glossary-container > dl > dt {
+  font-weight: 700;
+}
+#main-content .insider-perspective h3 {
```

Comment:
Which problem was this addressing? It looks like `.insider-perspective` is a book-specific style and the term "insider-perspective" does not occur anywhere in [webview](https://github.com/openstax/webview).

Maybe this was originally a bootstrap style that applies to all `<h3>` elements?